dev/functional/tests/cypress/page-objects/ministry-objects/StudentNoteRestrictionsObject.ts

```diff
@@ -7,6 +7,10 @@ export default class StudentNoteRestrictionsObject {
     return cy.contains("Add new restriction");
   }
 
+  waitForSec() {
```

Comment:
Do we really need this? It seems that we are just renaming a `cy` method from `focused` to `waitForSec`. Can you please elaborate on the benefits of doing this?

Reply:
Clicking the right element, not sure how to get around that "hidden" issue with cypress. cy.focus() can only be called on a valid focusable element.

Assertions:-
cy.focused() will automatically retry until the element(s) exist in the DOM
cy.focused() will automatically retry until all chained assertions have passed

Reply:
It was not the question or I am missing something. Why create a method called `waitForSec()` just for the sake of calling a single method `cy.focused()`?

Reply:
Will change this method name in next PR.

Reply:
Changes have been made.